src/index.js

```diff
@@ -2,33 +2,35 @@ import { getIdFromURL, getTimeFromURL } from './utils'
 import container from './container'
 import YouTubePlayer from './player'
 
-export { YouTubePlayer, getIdFromURL, getTimeFromURL }
+export default { YouTubePlayer, getIdFromURL, getTimeFromURL,
```

Comment:
Please revert the way of exporting.
This change is not necessary for the improvements, I think.